.github/workflows/multidim-interop.yml

```diff
@@ -12,3 +12,8 @@ jobs:
     uses: "./.github/workflows/run-testplans.yml"
     with:
       dir: "multidim-interop"
+  run-multidim-interop-ng:
```

Comment:
Should we just replace the above?

Reply:
I'm guessing the intent was to make sure the existing flow didn't break?

Reply:
Yeah, it is always easy enough to just remove the above later.